### DISCUSSION TITLE
Welcome to peaceful-pie Discussions!

### DISCUSSION THREAD

**Discussion by hughperkins** [hughperkins/peaceful-pie#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sutonpo** [comment]
I use Unity 2022.3.2 

I don't know how to find the log file path in Net Manager.

and I follow the tutorial, and I encounter the error message like

'payload {'method': 'Say', 'params': {'message': 'Hello'}, 'jsonrpc': '2.0', 'id': 0}
res <Response [404]>
res.content b'Malformed query: /jsonrpc'
e Expecting value: line 1 column 1 (char 0)
payload {'method': 'Say', 'params': {'message': 'Hello'}, 'jsonrpc': '2.0', 'id': 0}
res <Response [404]>'

how can I figure it out?

please help me, thanks!
